src/pages/Account.vue

```diff
@@ -25,6 +26,7 @@
                                 <div> {{ payment.pended_on }}</div>
                                 <div> {{ payment.vouchers.length }}</div>
                                 <div class="amount">£{{ payment.vouchers.length }}</div>
+                                <div class="download"><i class="fa fa-download" aria-hidden="true" title="Download"></i></div>
```

Comment:
When it's running we'll need to insert data based on payment.pended_on in here for vue wiring.